pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md

```diff
@@ -0,0 +1,58 @@
+### Version 6.10.0
+#### iOS Messaging SDK
+
+**Release Date**: October 07, 2022
+
+##### Environment requirements
+
+The iOS Mobile Messaging SDK version 6.10.0.849 is supported on iOS versions 13 through 16.
+
+**This XCFramework was compiled with Swift version Apple Swift version 5.5 (swiftlang-1300.0.31.1 clang-1300.0.29.1), which means it will work Swift version 5.5 and above.**
+
+{: .attn-alert}
+XCFramework is supported on CocoaPad versions 1.9.0 and greater.
+
+#### Bugs fixed
+
+- Cleared history is not coming back until user does re-login.
+- Phone numbers with spaces sent in messages are dial-able.
+- Fixed auto-layout constraints for in-app toast view to suppress warning being printed on XCode console.
+- Secure form message cell is fully accessible.
+- Convey the meaning of images through tooltip and the behavior if those are tap-able links within structured content.
+- Make a voice over announcement when a conversation ends by either consumer or agent.
+- Structured content elements font size will increase/decrease depending on font category.
+- Avoid welcome message being shown momentarily while loading conversation history.
+- Structured content text element is not cutting off content from trailing boundary.
+- Removing old proactive message from conversation when a new proactive message is opened.
+- LPMessagingSDKAgentDetails callback is being called when agent has changed to survey bot.
+- Set same height for structured content carousel buttons.
+- Stop clearing unauth user conversation history if another view covers conversation screen.
+- conversationBackgroundColor config being applied properly to the quick reply message cell.
+
+#### Enhancements
+
+- Based on a feature flag (by default false) images being sent to agents can be obfuscated for security reasons.
+- Improve the translations for strings and provide support for Filipino language.
+- Expose a config for setting minimum height of the structured content buttons.
+- Expose a config for setting structure content button border's corner radius.
+- Expose a config for setting structured content button of link type foreground color.
+
+#### New configurations
+
+##### structuredButtonMinimumHeight
+- **Description:** Set the minimum height constraint for Structured Constant button elements
+- **Type:** CGFloat?
+- **Default Value:**  nil
+- **Available since:** 6.10.0
+
+##### structuredContentButtonBorderCornerRadius
+- **Description:** Set the corner radius of Structured Constant button elements border
+- **Type:** CGFloat
+- **Default Value:**  15.0
+- **Available since:** 6.10.0
+
+##### structuredContentLinkButtonTextColor
```

Comment:
Suggestion: This looks like code, and if it is, consider backticks to mark it as code. Like:

```
##### `structuredContentLinkButtonTextColor`
````

Reply:
(Affects other headings, too.)

Reply:
Done.